packages/components/atoms/f-link/stories/Link.stories.js

```diff
@@ -40,21 +40,26 @@ export const VLinkComponent = () => ({
         noLineBreak: {
             default: boolean('noLineBreak', false)
         }
+    },
 
+    computed: {
+        target() {
+            console.log(this.opensInNewLocation ? '_blank' : null); // eslint-disable-line no-console
+            return this.opensInNewLocation ? '_blank' : null;
+        }
     },
```

Comment:
Do we need this? Since it's a `prop`.

Reply:
Definitely don't need the console log I left In.
I put it there to make it easier to test. Thought it would be helpful for the component testing using storybook. But can take it out if it's not necessary